packages/admin/admin/src/form/FieldContainer.tsx

```diff
@@ -125,6 +131,7 @@ export const FieldContainerComponent: React.FC<WithStyles<typeof styles> & Field
     if (fullWidth) formControlClasses.push(classes.fullWidth);
     if (hasError) formControlClasses.push(classes.hasError);
     if (hasWarning && !hasError) formControlClasses.push(classes.hasWarning);
+    if (helperText) formControlClasses.push(classes.helperText);
```

Comment:
I don't think this is correct: The helper text styles should only be applied to the helper text, but not the form control 🤔